proto/regen/ecocredit/v1/events.proto

```diff
@@ -106,6 +106,9 @@ message EventCancel {
 
   // amount is the decimal number of credits that have been cancelled.
   string amount = 3;
+
+  // reason is any arbitary encoded data.
+  string reason = 4;
```

Comment:
The issue and notes are a bit unclear on this but because we decided to add a specific event for bridging credits from regen ledger to another chain (`EventBridge`) and because this added event includes the necessary data we need for the bridge operation, this field will actually be used to provide a more generic reason.

We still want this field here because we are not storing a reason in state and this would be valuable information to have but the reason will not include encoded data such as the target chain, recipient, and contract. Also, because this is an event field, we don't need to include details about how the string needs to be formatted.

```suggestion
  // reason is the reason the credits were cancelled.
  string reason = 4;
```